### DISCUSSION TITLE
Видеозвонки

### DISCUSSION THREAD

**Discussion by kocik799** [mikopbx/Core#418] [Ideas (Идеи)]
Добрый день, имеется ли возможность совершать видеозвонки? или можно добавить такую возможность?

**Comment by boffart** [comment]
Такая возможность есть по умолчанию. 
Нужно разрешить видео кодеки:
<img width="1674" alt="MikoPBX_Системные_настройки" src="https://user-images.githubusercontent.com/5922739/195018842-be207c5a-1901-47d2-9862-a7df03da4d35.png">

**Reply by kocik799** [reply]
Отлично, спасибо.
